libs/SalesforceSDKCore/SalesforceSDKCoreTests/SFUserAccountManagerTests.m

```diff
@@ -86,15 +87,20 @@ - (void)setUp {
     // Delete the content of the global library directory
     NSString *globalLibraryDirectory = [[SFDirectoryManager sharedManager] directoryForUser:nil type:NSLibraryDirectory components:nil];
     [[[NSFileManager alloc] init] removeItemAtPath:globalLibraryDirectory error:nil];
-
     // Set the oauth client ID after deleting the content of the global library directory
     // to ensure the SFUserAccountManager sharedInstance loads from an empty directory
     self.uam = [SFUserAccountManager sharedInstance];
-    self.uam.oauthClientId = @"fakeClientIdForTesting";
+    [SFAuthenticationManager sharedManager].oauthClientId = @"fakeClientIdForTesting";
 
     // Ensure the user account manager doesn't contain any account
+    NSArray *userAccounts = [[SFUserAccountManager sharedInstance] allUserAccounts];
```

Comment:
`[SFUserAccountManager clearAllAccountState]` (called below) is a unit-test-specific method that's basically meant to accomplish what this is doing.  Should we just update that method to physically remove accounts as part of its process, and remove this code?